go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/openshift/installer
 go 1.14
 
 require (
-	cloud.google.com/go v0.57.0
+	cloud.google.com/go v0.65.0
```

Comment:
This bump is not supported by the gcp terraform provider.

Reply:
This could explain the failures I was seeing :)